app/policies/security_question_policy.rb

```diff
@@ -0,0 +1,16 @@
+class SecurityQuestionPolicy < ApplicationPolicy
+
+  def edit?
+    has_security_questions?
+  end
+
+  def update?
+    has_security_questions?
+  end
+
+  private
+
+  def has_security_questions?
+    user && !user.portal_student.nil?
+  end
```

Comment:
Ok.  I tried to mirror the old method names in the policies - that is why it has that name.  The old controller had:

    before_filter :user_has_security_questions

    def user_has_security_questions
      unless current_visitor && !current_visitor.portal_student.nil?
        redirect_to(root_path)
        return
      end
    end